python/requirements-wheel-test.txt

```diff
@@ -7,18 +7,23 @@ pytz
 tzdata; sys_platform == 'win32'
 
 numpy==1.21.3; platform_system == "Linux"   and platform_machine == "aarch64" and python_version < "3.11"
-numpy==1.23.4; platform_system == "Linux"   and platform_machine == "aarch64" and python_version >= "3.11"
+numpy==1.23.4; platform_system == "Linux"   and platform_machine == "aarch64" and python_version == "3.11"
+numpy==1.26.0; platform_system == "Linux"   and platform_machine == "aarch64" and python_version >= "3.12"
 numpy==1.19.5; platform_system == "Linux"   and platform_machine != "aarch64" and python_version <  "3.9"
 numpy==1.21.3; platform_system == "Linux"   and platform_machine != "aarch64" and python_version >= "3.9" and python_version < "3.11"
-numpy==1.23.4; platform_system == "Linux"   and platform_machine != "aarch64" and python_version >= "3.11"
+numpy==1.23.4; platform_system == "Linux"   and platform_machine != "aarch64" and python_version == "3.11"
+numpy==1.26.0; platform_system == "Linux"   and platform_machine != "aarch64" and python_version >= "3.11"
 numpy==1.21.3; platform_system == "Darwin"  and platform_machine == "arm64"   and python_version <  "3.11"
-numpy==1.23.4; platform_system == "Darwin"  and platform_machine == "arm64"   and python_version >= "3.11"
+numpy==1.23.4; platform_system == "Darwin"  and platform_machine == "arm64"   and python_version == "3.11"
+numpy==1.26.0; platform_system == "Darwin"  and platform_machine == "arm64"   and python_version >= "3.12"
 numpy==1.19.5; platform_system == "Darwin"  and platform_machine != "arm64"   and python_version <  "3.9"
 numpy==1.21.3; platform_system == "Darwin"  and platform_machine != "arm64"   and python_version >= "3.9" and python_version < "3.11"
-numpy==1.23.4; platform_system == "Darwin"  and platform_machine != "arm64"   and python_version >= "3.11"
+numpy==1.23.4; platform_system == "Darwin"  and platform_machine != "arm64"   and python_version == "3.11"
+numpy==1.26.0; platform_system == "Darwin"  and platform_machine != "arm64"   and python_version >= "3.12"
 numpy==1.19.5; platform_system == "Windows"                                   and python_version <  "3.9"
 numpy==1.21.3; platform_system == "Windows"                                   and python_version >= "3.9" and python_version < "3.11"
-numpy==1.23.4; platform_system == "Windows"                                   and python_version >= "3.11"
+numpy==1.23.4; platform_system == "Windows"                                   and python_version == "3.11"
+numpy==1.26.0; platform_system == "Windows"                                   and python_version >= "3.12"
```

Comment:
Why are you we differentiating by platform for Python 3.11 and 3.12? It looks like this could be simplified.

Reply:
The question is also why we have this version pinning at all. We can just let pip install the latest available version for the specific python version it is installing for.

(but that's for a different PR)

Reply:
Because we want to test that the wheel is ABI-compatible with the oldest possible Numpy versions (as limited by Python version and platform).

Reply:
Ah, yes (I was assuming we didn't want to test that aspect here, because it are not _exactly_ the oldest versions for some python versions, eg for Python 3.9. But for testing that we properly use oldest-supported-numpy, just testing some older version is probably good enough). 
But I think we can certainly simplify this to not differentiate between platforms (and also for pandas that shouldn't matter though). Will open an issue about it.